config/locales/en.yml

```diff
@@ -2425,10 +2425,10 @@ en:
         filter_value_bands:
           all: All
           value: 'Value'
-          tier1: less than £25,000
-          tier2: £25,000 - £100,000
-          tier3: £100,000 - £150,000
-          tier4: more than £150,000
+          tier1: less than £30,000
```

Comment:
sorry to nitpick, i'm assuming this should be `£30,000 or less` or are we preferring this way for users readability?

Reply:
I think your nitpick is technically correct, but `less than £xx,xxx` is the current wording which caseworkers are used to so it seems sensible to leave it as it is. Also the chances of receiving a claim that is exactly £30,000.00 is so small that we probably don't need to worry about it.